qfdmo/views/adresses.py

```diff
@@ -587,6 +587,10 @@ def refresh_acteur_view(request):
 
 @require_GET
 def get_object_list(request):
+    q = request.GET.get("q")
+    if not q:
+        return JsonResponse([], safe=False)
+
     query = unidecode.unidecode(request.GET.get("q"))
```

Comment:
```suggestion
    query = unidecode.unidecode(q)
```

Reply:
ou pour suivre le reste de la recommander 

```suggestion
    query = unidecode.unidecode(query)
```